src/current/v24.1/molt-fetch.md

```diff
@@ -312,6 +429,59 @@ To retry all data starting from a specific file, include both `--fetch-id` and `
 Continuation is not possible when using [direct copy mode](#direct-copy).
 {{site.data.alerts.end}}
 
+#### List active continuation tokens
+
+To view all active continuation tokens, issue a `molt fetch token` command along with `--conn-string`, which specifies the [connection string]({% link {{ page.version.version }}/connection-parameters.md %}#connect-using-a-url) for the target CockroachDB database. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+molt fetch token \
+--conn-string 'postgres://root@localhost:26257/defaultdb?sslmode=verify-full'
+~~~
+
+~~~
++--------------------------------------+--------------------------------------+------------------+----------------------+
+|                  ID                  |               FETCH ID               |    TABLE NAME    |      FILE NAME       |
++--------------------------------------+--------------------------------------+------------------+----------------------+
+| f6f0284c-d9c1-43c9-8fde-af609d0dbd82 | 66443597-5689-4df3-a7b9-9fc5e27180eb | public.employees | part_00000001.tar.gz |
++--------------------------------------+--------------------------------------+------------------+----------------------+
+Continuation Tokens.
+~~~
+
+### Replication
+
+`--ongoing-replication` enables logical replication from the source database to the target CockroachDB database. 
+
+{% include_cached copy-clipboard.html %}
+~~~
+--ongoing-replication
+~~~
+
+When the `--ongoing-replication` flag is set, changes on the source database are continuously replicated on CockroachDB. This begins only after the fetch process succeeds—i.e., the initial source data is loaded into CockroachDB—as indicated by a `fetch complete` message in the output.
+
+Before using this feature, complete the following:
+
+- Install the Replicator binary. Before running `molt fetch` with continuous replication, [download the binary that matches your system](https://github.com/cockroachdb/replicator/wiki/Installing#automated-builds). The Replicator binary **must** be located in the same directory as your [`molt` binary](#installation).
+- Configure the source PostgreSQL or MySQL database for continuous replication, as described in [Setup](#setup).
+
+If the source is a PostgreSQL database, you must also specify a replication slot name:
+
+{% include_cached copy-clipboard.html %}
+~~~
+--ongoing-replication
+--pglogical-replication-slot-name 'replication_slot'
+~~~
+
+If you need to customize the Replicator behavior, use `--replicator-flags` to specify one or more Replicator flags ([PostgreSQL](https://github.com/cockroachdb/replicator/wiki/PGLogical#postgresql-logical-replication) or [MySQL](https://github.com/cockroachdb/replicator/wiki/MYLogical#mysqlmariadb-replication)) to override. This will only be necessary for advanced use cases.
+
+{% include_cached copy-clipboard.html %}
+~~~
+--ongoing-replication
+--replicator-flags '--applyTimeout 1h'
```

Comment:
I think we might want to show the example of more than 1 flag.
cc @ryanluu12345 to provide example.

Reply:
`--replicator-flags "--applyTimeout '1h' --parallelism 64`